### PR TITLE
Fix config files for private chain tests

### LINKED ISSUE #4303: Fix test pipelines
Nightly pipelines for dev are broken since few weeks now and could use some care.

### PR DESCRIPTION
**Pull request by rafal-ch**
This PR updates the test config files for the NCTL private chain tests. It also removes the stray test files that had been unused.

Closes #4303

### CODE REVIEW

File: utils/nctl/sh/scenarios/itst01_private_chain.sh
Comment (alsrdn) [inline utils/nctl/sh/scenarios/itst01_private_chain.sh:1]: this links to an absolute path on your machine. Can you fix it to link to a relative path?

File: utils/nctl/sh/scenarios/itst02_private_chain.sh
Comment (alsrdn) [inline utils/nctl/sh/scenarios/itst02_private_chain.sh:1]: same as above

File: utils/nctl/sh/scenarios/accounts_toml/itst01_private_chain.accounts.toml.override [hunk 1]
```diff
@@ -43,6 +43,12 @@ balance = "0"
 bonded_amount = "1000000000000005"
 delegation_rate = 5
 
+[[delegators]]
+validator_public_key = "PBK_V5"
+delegator_public_key = "020248509e67db3127f82d5224c5c18eac00f96d1edeadbadc8eb2c8606227b56873"
```
Comment (alsrdn) [inline utils/nctl/sh/scenarios/accounts_toml/itst01_private_chain.accounts.toml.override]: It's unclear who the delegator is here. We should use symbolic names in the overrides e.g. `PBK_U1`.

File: utils/nctl/sh/scenarios/accounts_toml/itst02_private_chain.accounts.toml.override [hunk 2]
```diff
@@ -43,6 +43,12 @@ balance = "0"
 bonded_amount = "1000000000000005"
 delegation_rate = 5
 
+[[delegators]]
+validator_public_key = "PBK_V5"
+delegator_public_key = "020248509e67db3127f82d5224c5c18eac00f96d1edeadbadc8eb2c8606227b56873"
```
Comment (alsrdn) [inline utils/nctl/sh/scenarios/accounts_toml/itst02_private_chain.accounts.toml.override]: same as above

**Review by alsrdn — CHANGES_REQUESTED** [review]

File: utils/nctl/sh/scenarios/itst01_private_chain.sh
Comment (rafal-ch) [reply utils/nctl/sh/scenarios/itst01_private_chain.sh:1]: Good catch! :+1: 
Fixed in https://github.com/casper-network/casper-node/pull/4343/commits/eec2d324a4416c6d1276a9e21fb42645b8f84ab9

File: utils/nctl/sh/scenarios/itst02_private_chain.sh
Comment (rafal-ch) [reply utils/nctl/sh/scenarios/itst02_private_chain.sh:1]: same as above ;-)

File: utils/nctl/sh/scenarios/accounts_toml/itst01_private_chain.accounts.toml.override (hunk 1 above)
Comment (rafal-ch) [reply utils/nctl/sh/scenarios/accounts_toml/itst01_private_chain.accounts.toml.override]: Fixed, along with other necessary changes, in https://github.com/casper-network/casper-node/pull/4343/commits/f6af9d9b623b9669f92398c2ab14b105abc834d0

File: utils/nctl/sh/scenarios/accounts_toml/itst02_private_chain.accounts.toml.override (hunk 2 above)
Comment (rafal-ch) [reply utils/nctl/sh/scenarios/accounts_toml/itst02_private_chain.accounts.toml.override]: Fixed, along with other necessary changes, in https://github.com/casper-network/casper-node/pull/4343/commits/f6af9d9b623b9669f92398c2ab14b105abc834d0

**Comment by rafal-ch** [comment]
bors try

**Review by alsrdn — APPROVED** [review]
LGTM

**Review by mpapierski — APPROVED** [review]

**Pull request merged**